extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeSubscriptionExtractor.java

```diff
@@ -68,4 +97,109 @@ public String getRelatedUrl() {
         }
         return subscriptionItems;
     }
+
+    public List<SubscriptionItem> fromZipInputStream(@Nonnull final InputStream contentInputStream)
+            throws ExtractionException {
+        final ZipInputStream zipInputStream = new ZipInputStream(contentInputStream);
```

Comment:
This zipInputStream is not closed anywhere. You should close the stream after its use.

Reply:
Oh, right, didn't notice this

Reply:
@Stypox Hotfix?

Reply:
Nope, this was not released. We just need a PR with a try-with-resources

Reply:
Oh. Yeah. I mixed up the PRs.

Reply:
Thank you @Sanjay15693 :-)
See #721